.github/workflows/tests.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: make bootstrap
 
       - name: Lint
-        run: make lint GOLANGCI_LINT_FORMAT=github-actions
+        run: make lint GOLANGCI_LINT_FORMAT=colored-line-number
```

Comment:
Is this change required?

Reply:
I think it still works with a warning, but the "github-actions" format was deprecated in [v1.59.0](https://github.com/golangci/golangci-lint/releases/tag/v1.59.0)